### ISSUE TITLE
resnet

### ISSUE THREAD

**Issue by nationalflag** [yjxiong/tsn-pytorch#7]
Will the results become better by using the resnet101 as pretrained model?

**Comment by yjxiong** [comment]
According to our experiences, deeper CNNs lead to better performance. You can try switching the architecture to resnet101 to test its performance by setting 
` --arch resnet101`

**Comment by AlexHu123** [comment]
hi, yjxiong, would it be possible for you to tell me the command you use to train resnet-50? I use the default command in the readme, but get only 78% accuracy.
